src/SignUpIn/SignUpStudentController.js

```diff
@@ -0,0 +1,135 @@
+// NavBarButton.js
+// import React, { Component } from 'react';
+import logo from 'public-encrypt';
+import './SignUpIn.css';
+// import './SignUpStudent';
+import StudentSignUp from './StudentSignUp';
+import StudentSignUp1 from './StudentSignUp1';
+import StudentSignUp2 from './StudentSignUp2';
+import StudentSignUp3 from './StudentSignIn3';
+import firebase from 'firebase/app'
+let React = require('react');
+let createReactClass = require('create-react-class');
+
+
+// class SignUpStudent extends Component {
+// constructor(props, context){
+//     super(props, context)
+//     this.state = {
+//         step: 1
+//     }
+let fieldValues = {
+    organizationName: null,
+    numVolunteers: null,
+    contactName: null,
+    contactEmail: null,
+    contactNumber: null,
+    email: null,
+    password: null,
+    memberName: null,
+    memberNumber: null,
+    memberEmail: null,
+    position: null
+}
+
+let SignUpStudentController = createReactClass({
+    getInitialState: function () {
+        return {
+            step: 0
+        }
+    },
+    saveValues: function (fields) {
+        return function () {
+            // Remember, `fieldValues` is set at the top of this file, we are simply appending
+            // to and overriding keys in `fieldValues` with the `fields` with Object.assign
+            // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign
+            fieldValues = Object.assign({}, fieldValues, fields)
+        }()
+    },
+
+    nextStep: function () {
+        this.setState({
+            step: this.state.step + 1
+        })
+    },
+
+    previousStep: function () {
+        this.setState({
+            step: this.state.step - 1
+        })
+    },
+
+    submitRegistration: function () {
+        // Handle via ajax submitting the user data, upon
+        // success return this.nextStop(). If it fails,
+        // show the user the error but don't advance
+        
+        firebase.auth().createUserWithEmailAndPassword(fieldValues.email, fieldValues.password)
+            .then(user => {
+                console.log('User created: ' + user.uid);
+                let postData = {
```

Comment:
Please follow the schema in `src/schema.js` or in the "Backend" doc on the drive. In this case, you are creating a DelivererGroup object. Update/add fields as you see fit, but please follow the documentation and keep it updated.